src/components/SearchResults.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import "../styles/SearchResults.css";
+
+const SearchResults = ({ results, userHasSearched }) => {
+  if (results.length === 0 && userHasSearched === false) {
+    return null;
+  }
```

Comment:
This block isnt really doing anything, you're accounting for both behaviours of the component (show the images or the none found message) below so this can be removed